src/kv/kv.h

```diff
@@ -1001,7 +1001,7 @@ namespace kv
         auto h = store->get_history();
         if (h != nullptr)
         {
-          h->add_result(req_id, version);
+          h->add_result(req_id, NoVersion);
```

Comment:
Why is the NoVersion?

Reply:
This is for a read only tx so there is no new version of the KV. I am using the NoVersion to specify that we do not need to compact

Reply:
Could you add a comment to document this assumption please?